java/src/jmri/jmrit/signalling/AddEntryExitPairPanel.java

```diff
@@ -135,10 +135,14 @@ private void addButton() {
 
     private void autoDiscovery() {
         if (!InstanceManager.getDefault(jmri.jmrit.display.layoutEditor.LayoutBlockManager.class).isAdvancedRoutingEnabled()) {
-            int response = JOptionPane.showConfirmDialog(null, Bundle.getMessage("EnableLayoutBlockRouting"));
+            int response = JOptionPane.showConfirmDialog(null,
```

Comment:
I could do the merge.  I created a spreadsheet of the keys.  There is one duplicate key.

The Catalan, Danish, French and Italian translations are missing some keys.  The French version of one bundle does not exist.

What is the best way to deal with the missing keys?  Don't include them, include them but commented, or something else?